src/main/resources/profiles/digital-commonwealth.conf

```diff
@@ -11,6 +11,5 @@ endpoint = "http://fedora.digitalcommonwealth.org/oaiprovider/"
 metadataPrefix = "mods"
 outputDir = "" # TODO This should be an S3:// bucket that i3 has write access to
 provider = "Digital Commonwealth"
-type = "oai" # Which harvester to invoke
```

Comment:
Since the scope of the PR is to update just the README, can you comment on the removal of this line and the one in `oaiSample.conf`?